### PR TITLE
Protocol Type

### PR DESCRIPTION
**Pull request by Hirobreak**
only process messages when protocol type is 1 or 2

### CODE REVIEW

**Review by danieltigse — APPROVED** [review]

File: main.js [hunk 1]
```diff
@@ -789,7 +789,7 @@ require('es6-promise').polyfill();
       message.readByUser = true;
     }
 
-    if(message.id !== "0"){
+    if(message.id !== "0" && message.protocolType < 3){
```
Comment (GianniCarlo) [inline main.js:792]: Magic numbers @Hirobreak

Comment (danieltigse) [reply main.js:792]: You are on vacations man. Enjoy. But your are right! Magic numbers

**Comment by GianniCarlo** [comment]
Sussy is resting before returning to the parks, killing time catching up (got nothing else to do 😞 )

File: main.js (hunk 1 above)
Comment (danieltigse) [reply main.js:792]: Oh OK I would do the same haha

**Pull request merged**